### ISSUE TITLE
question about docker

### ISSUE THREAD

**Issue by StevenChorkley** [mdhiggins/sickbeard_mp4_automator#1227]
Hi Mate, 

Apologies, I know you're not going to appreciate the question - but I've never been able to figure out the answer. 

In regards to the docker containers, why don't you handle the conversions in nzbget, rather than sonarr/radarr respectively? 
Personally I prefer the nzbget conversions because you can easily see the conversion progression. 

Additionally - If you run the script in NZBGET, is there any need to run the script again in Sonarr/Radarr? 

Again, sorry for asking :P

**Comment by mdhiggins** [comment]
No big deal, don't mind questions

Soanrr/Radarr pass the metadata ID for the files so I use that to pull the tagging metadata. As a result the tagging step is only done at that level (not with NZBGet or other downloaders). The only place where a single conversion / tagging can take place is at the media manager level.

As far as I know, NZBGet doesn't actually report conversion progress (I've pushed for them to add this to their logging for a while) for post scripts, its just a generic progressbar that's not in sync with the conversion progress at all.

I maintain the 2 dockers because that's what I use in my personal setup and its the path of least resistance and suits my needs. I've worked with others to develop additional containers but just put them out there to help others make their setup easier and to serve as a template to build additional containers if someone desired. 

That's at least the reasoning behind it. I'm sure others have made SMA containing dockers I certainly wasn't the first. I was kind of late to the docker game.
